javascript/src/main/pages/AnalyzeMessageData/SearchBetweenDates.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState} from "react";
+import { useAuth0 } from "@auth0/auth0-react";
+import { fetchWithToken } from "main/utils/fetch";
+import { Form, Button } from "react-bootstrap";
+import MessageListView from "main/components/ChannelMessages/MessageListView"
+
+const SearchBetweenDates = () => {
+    const { getAccessTokenSilently: getToken } = useAuth0();
+    const [ searchDate, setSearchDate ] = useState(''); 
+    const [ searchDate2, setSearchDate2 ] = useState('');
+    const [ searchResults, setSearchResults ] = useState([]);
+
+    const handleSearchDateOnChange = (event) => {
+        setSearchDate((new Date(event.target.value).getTime() / 1000).toString());
+    };
+
+    const handleSearchDateOnChange2 = (event) => {
+        setSearchDate2((new Date(event.target.value).getTime() / 1000).toString());
+    };
+    const handleSearchDateOnSubmit = (event) => {
+        event.preventDefault();
+        const url = `/api/members/messages/datesearch?searchDate=${searchDate}&searchDate2=${searchDate2}`;

```

Comment:
I don't really like the names `searchDate` and `searchDate2` - it's kinda unclear what these are at first glance. I think `startDate` and `endDate` would be better names - I can see them being used as the form labels, so there isn't any reason that I'm aware of that prevents us from using these variable names in our code. Can we make this change everywhere?